tools/azure-sdk-tools/ci_tools/build.py

```diff
@@ -179,10 +179,11 @@ def create_package(
     dist = get_artifact_directory(dest_folder)
     setup_parsed = ParsedSetup.from_path(setup_directory_or_file)
 
-    if setup_parsed.ext_modules:
-        run([sys.executable, "-m", "cibuildwheel", "--output-dir", dist], cwd=setup_parsed.folder, check=True)
-
     if enable_wheel:
-        run([sys.executable, "setup.py", "bdist_wheel", "-d", dist], cwd=setup_parsed.folder, check=True)
+        if setup_parsed.ext_modules:
```

Comment:
To the viewers of this PR. This is a straight up bugfix. Prior to this, for extension packages were _also_ building a standard wheel. It should be one or the other!